Remotion/ObjectBinding/Web/ObjectBinding.Web.csproj

```diff
@@ -18,6 +18,9 @@
       <PackagePath>res/HTML</PackagePath>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <None Update="..\..\Remotion LGPL.licenseheader" Pack="true" PackagePath="res/HTML" />
```

Comment:
We should change the package path to `res/` in all of the cases. I don't think we have any other files with licenseheaders in any other folder than `HTML` but it can't hurt.